cli/devlog.py

```
@@ -312,8 +306,140 @@
+        all_site_success = all(site_results.values()) if site_results else False
+        all_blog_success = all(blog_results.values()) if blog_results else False
+        
+        if all_site_success and all_blog_success:
+            click.echo("[OK] All files published successfully")
+        else:
+            click.echo("[WARN] Some files failed to publish")
+            if not all_site_success:
+                click.echo("[ERR] Site publishing had failures")
+            if not all_blog_success:
+                click.echo("[ERR] Blog publishing had failures")
+            raise SystemExit(1)
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)